### PR TITLE
[full-ci] Update Reva and add Denials config

### PR DESCRIPTION
**Pull request by micbar**
## Deny ACLs

Update reva to test https://github.com/cs3org/reva/pull/3011

## Motivation and Context
The deny feature will be disabled by default for the GA release and marked as `EXPERIMENTAL`

Two config vars are added to enable the feature both on WebDAV and OCS Api.

The permission is announced by the WebDAV Permission `Z` which means you can set a  full denial on the resource.

## Capability

we added a new capability `deny_access` in the `files_sharing` section.

## How Has This Been Tested?
- CI
- manually

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [x] Unit tests added (reva side)
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by phil-davis — APPROVED** [review]

File: services/frontend/pkg/config/config.go [hunk 1]
```diff
@@ -117,6 +117,7 @@ type OCS struct {
 	ResourceInfoCaches      ResourceInfoCaches `yaml:"resource_info_caches,omitempty"` // only used for redis
 	CacheWarmupDriver       string             `yaml:"cache_warmup_driver,omitempty"`  // not supported by the oCIS product, therefore not part of docs
 	CacheWarmupDrivers      CacheWarmupDrivers `yaml:"cache_warmup_drivers,omitempty"` // not supported by the oCIS product, therefore not part of docs
+	EnableDenials           bool               `yaml:"enable_denials" env:"FRONTEND_OCS_ENABLE_DENIALS" desc:"EXPERIMENTAL: enable the option to deny access on resources."`
```
Comment (mmattel) [inline services/frontend/pkg/config/config.go:120]: Reading that this is experimental - ok...
But:  `enable the option to deny access on resources` is very cryptic.
- `Enable the option` means for me, that there is another value that can be manipulated that does the details.
- `Ressources`, hmmm, any hint to be added what is meant, or an example?

Comment (micbar) [reply services/frontend/pkg/config/config.go:120]: > Ressources, hmmm, any hint to be added what is meant, or an example?

Resource is used all over the code base and means "file or folder"

> Enable the option

We can say `enable the feature`

**Comment by micbar** [comment]
@kulmann FYI, this includes the new capability.

**Comment by kulmann** [comment]
> @kulmann FYI, this includes the new capability.

Thanks! Created https://github.com/owncloud/web/pull/7892 for respecting the capability in web.

**Pull request merged**